### ISSUE TITLE
Eslint import error with v4.0.0

### ISSUE THREAD

**Issue by af** [rotaready/moment-range#244]
After upgrading from 3.1.1 from 4.0.0, I get the following eslint error when importing `moment-range`, via `eslint-plugin-import`:

```
12:28  error  Unable to resolve path to module 'moment-range'  import/no-unresolved
```

I've triple checked and the correct package is in my `yarn.lock`, `package.json`, and `node_modules/`. I can't see any changes in `moment-range`'s package.json since 3.1.1 that would cause this error either. Has anyone else experienced this?

**Comment by gf3** [comment]
perhaps eslint now requires an extension for the `main` key? https://github.com/rotaready/moment-range/blob/f096722c51777185b9ad1ce11cc75c007098b3a2/package.json#L29

**Comment by gf3** [comment]
@af can you paste your eslint config here and i'll try to reproduce

**Comment by mlandalv** [comment]
I have the same problem. The reason is the "jsnext:main" key in packages.conf.

    "jsnext:main": "lib/moment-range.js",

However, in moment-range v4 the lib folder isn't published, only the dist folder, so jsnext:main points to a non-existing file, hence the eslint error.

According the the v4 changelog all lib-entries should have been removed from `packages.conf`, so I'm guessing this one was missed. `jsnext:main` should simply be removed.

**Comment by gf3** [comment]
ahhh thanks @mlandalv

**Comment by TristanJM** [comment]
Thanks to @mlandalv we've got this released as version `4.0.1`

Let me know if the problem is now fixed @af and we can close off this issue 😃

**Comment by af** [comment]
Looks like this is resolved on my end. Thank you!
